### PR TITLE
Server: send prompt periodically

### PR DESCRIPTION
**Pull request by fjebaker**

### CODE REVIEW

File: server/connections.py [hunk 1]
```diff
@@ -143,25 +149,39 @@ def __init__(self, prompts: list[tuple[str, list[str]]]):
         self.usermap: dict[int, UserConnection] = {}
         self.team = {1: [], 2: []}
         self.score = {1: 0, 2: 0}
-        self.buffers: dict[int, list[dict]] = {1: [], 2: []}
+        self._init_empty_buffers()
         self.prompts = prompts
         self.current_prompt : tuple[str, list[str]] | None = None
 
+    def _init_empty_buffers(self) -> None:
```
Comment (JR-Mitchell) [inline server/connections.py:156]: nice refactoring

File: server/connections.py [hunk 2]
```diff
@@ -143,25 +149,39 @@ def __init__(self, prompts: list[tuple[str, list[str]]]):
...
         return len(self.usermap) == 4
 
     async def start(self) -> None:
         logger.info("Starting round")
+        # clear whatever is in the current buffers
+        self._init_empty_buffers()
 
         self.current_prompt = random.choice(self.prompts)
-        key, hints = self.current_prompt[0], self.current_prompt[1:]
+        key, hints = self.current_prompt
```
Comment (JR-Mitchell) [inline server/connections.py:168]: I don't see where the type has changed, can you explain this change to me?

Comment (fjebaker) [reply server/connections.py:168]: Tuple only has 2 elements

**Review by JR-Mitchell — APPROVED** [review]
:chipmunk: :computer:

**Pull request merged**